src/Run/Setup.cs

```diff
@@ -271,14 +268,14 @@ private string GetTool(Command commandToExecute, string os, string configPath, L
             if (Tools.ContainsKey(toolname))
```

Comment:
True, a while ago I did that refactoring but it seems I missed this one. I'll take a look at the code to see where else I have this conditions and I'll change it.